recipe/meta.yaml

```diff
@@ -1,45 +1,50 @@
 {% set name = "nvidia-ml" %}
-{% set pypi_name = "nvidia-ml-py3" %}
-{% set version = "7.352.0" %}
+{% set pypi_name = "nvidia-ml-py" %}
+{% set version = "11.525.112" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ pypi_name }}/{{ pypi_name }}-{{ version }}.tar.gz
-  sha256: 390f02919ee9d73fe63a98c73101061a6b37fa694a793abf56673320f1f51277
+  sha256: c64e473953b65ec0ccc76cf3601c3130a0ac8060bbcaea912cc00fa8e4c9868d
 
 build:
   noarch: python
   number: 0
-  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   host:
     - python
     - pip
+    - setuptools
+    - wheel
   run:
     - python
```

Comment:
It has `CUDA` support, see the **archlinux** build recipe https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=python-nvidia-ml-py#n12. Do we need it enabled? But I'm not sure what purpose `--optimise=1` and `--skip-build` are used for https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=python-nvidia-ml-py#n29

However the upstream `setup.py` from [the source file](https://files.pythonhosted.org/packages/f4/e9/8279f2dbecbd22433ba2ba997688873b5131cb4d28b0381f86a4701cd8b1/nvidia-ml-py-11.525.112.tar.gz) doesn't have `install_requires`:

```
setup(name=_package_name,
      version='11.525.112',
      description='Python Bindings for the NVIDIA Management Library',
      long_description=long_description,
      long_description_content_type='text/markdown',
      py_modules=['pynvml', 'example'],
      package_data={_package_name: ['Example.txt']},
      license='BSD',
      url='https://forums.developer.nvidia.com',
      author='NVIDIA Corporation',
      author_email='nvml-bindings@nvidia.com',
      classifiers=[
          'Development Status :: 5 - Production/Stable',
          'Intended Audience :: Developers',
          'Intended Audience :: System Administrators',
          'License :: OSI Approved :: BSD License',
          'Operating System :: Microsoft :: Windows',
          'Operating System :: POSIX :: Linux',
          'Programming Language :: Python',
          'Topic :: Software Development :: Libraries :: Python Modules',
          'Topic :: System :: Hardware',
          'Topic :: System :: Systems Administration',
          ],
      )
```